sdcm/provision/common/utils.py

```diff
@@ -120,6 +120,14 @@ def configure_sshd_script():
     """)
 
 
+def configure_legacy_rsa_keys_for_sshd():
```

Comment:
This commit doesn't have any description and I couldn't understand why it was needed without reading the comments here.
Please explain the requirement and the fact that it affects all distros.

In addition, the title is wrong, it shouldn't be "fix(nemesis)".